manifests/charts/istio-cni/templates/configmap-cni.yaml

```diff
@@ -30,13 +30,16 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: "istio-cni-taint-configmap"
-  namespace: "kube-system"
+  namespace: {{ .Release.Namespace }}
```

Comment:
Looks like if we update the namespace to `{{ .Release.Namespace }}`, the operator will deploy this into `istio-system` namespace by default, is this intent?

Reply:
Yes. I don't see why this configmap should be deployed to kube system while other manifests of istio cni stay in Istio system. It feels a bit off